src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.ecommercee.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+public class Group {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(unique = true)
+    private Long id;
+
+    @Column
+    private String name;
+
+    @JsonIgnore
```

Comment:
JSON to będzie bardziej przy obiektach DTO.

Reply:
Ok